src/cli/BaseCLIController.ts

```diff
@@ -88,4 +89,18 @@ export class BaseCLIController {
     const { error } = await this.execDvc('--version')
     return !error
   }
+  
+  public requiredPackageVersion = '5.2.1'
```

Comment:
it'd be nice if we could surface this value to somewhere in the root of the repo so that it's more visible and easier to update

Reply:
yeah I was thinking this as well. Maybe just have a `requirements.json` file or something?